fallout-tests/dse47-zdmproxy-benchmark-nb-vs-noproxy-astra-target.yaml

```diff
@@ -0,0 +1,350 @@
+origin_server_product: dse
+origin_server_install_type: tarball
+origin_server_version: 4.7.9
+origin_server_tar_download_link: https://downloads.datastax.com/enterprise/dse-4.7.9-bin.tar.gz
+origin_server_linux_distro: bionic
+origin_node_count: 3
+
+target_astra_service_account: prod_wei
+target_astra_write_rate: 9000
+
+proxy_branch: main
+proxy_node_count: 3
+instance_type: m5.large
+
+read_mode: PRIMARY_ONLY
+primary_cluster: ORIGIN
+replace_cql_functions: false
+enable_profiling: false
+
+rampup_cycles: 10000
+main_cycles: 100000
+num_threads: 128
+num_connections: 8
+
+enable_metrics: true
+
+activity_repo: riptano/data-model-archive
+activity_repo_branch: master
+activity_path: astra/baselines-astra/activities
+activity_file: cql-keyvalue-astra.yaml
+
+ttl: 2h
+reuse: false
+
+---
+ensemble:
+  servers:
+    - name: origin
+      node.count: {{origin_node_count}}
+      provisioner:
+        name: ctool
+        properties:
+          cloud.provider: ec2
+          cloud.instance.type: {{instance_type}}
+          cloud.instance.platform: {{origin_server_linux_distro}}
+          mark_for_reuse: {{reuse}}
+          cluster_ttl: {{ttl}}
+      configuration_manager:
+        - name: ctool
+          properties:
+            product.type: {{origin_server_product}}
+            product.install.type: {{origin_server_install_type}}
+            product.version: {{origin_server_version}}
+            product.tar.url: {{origin_server_tar_download_link}}
+            enable.graph: false
+            datacenters:
+              dc_origin:
+                size: {{origin_node_count}}
+                workload: cassandra
+    - name: target
+      node.count: 1
+      provisioner:
+        name: astra
+        properties:
+          service_account: {{target_astra_service_account}}
```

Comment:
Remove this on all the astra tests so it uses the default account instead.

I couldn't import the yaml as is because of the service account name.

Reply:
good catch. Let me fix that.